### PR TITLE
Added prefference menu for easier configuration from within Sublime.

### PR DESCRIPTION
**Pull request by brushesm**
Hi, that's the pull request with my changes.

### CODE REVIEW

File: helpdb/esmscripten.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (Swoffa) [inline helpdb/esmscripten.json]: Is this Kick Assembler related things?

Comment (taugustyn) [reply helpdb/esmscripten.json]: That should be removed @brushesm ;-)

**Comment by brushesm** [comment]
You are right. I will correct it.

**Comment by Swoffa** [comment]
I am very excited to get more action in this package. THANK YOU! :)
Debugger stuff, very interesting! 
I would suggest one branch/pull request per "feature" (tooltips, debugger, preference menu etc.) if possible. Easier to overlook, handle and release. Is it possible?

**Comment by brushesm** [comment]
Let's do the following: i will remove the unnecesary file and please merge tooltips + preference menu. The debugger stuff i will do in branch.

**Comment by Swoffa** [comment]
> Let's do the following: i will remove the unnecesary file and please merge tooltips + preference menu. The debugger stuff i will do in branch.
Sounds good.

**Comment by brushesm** [comment]
Please delete this pull request. I made a new fork and i will make a clean work for you.
